our-process/policies.md

```diff
@@ -24,8 +24,8 @@ When submitting a package, please make sure that your GitHub notification settin
 
 We strongly suggest submitting your package for review with pyOpenSci before:
 
-*  publishing on `PyPI` or `conda`;
-*  submitting a software paper describing the package to a journal.
+- publishing on `PyPI` or `conda`;
```

Comment:
I know this is just a formatting change, but should it say 
```suggestion
- publishing on `PyPI` (and/or `conda`);
```

Reply:
Do you anticipate people to do this on every publish of a release to PyPI or `conda-forge`? Major releases or just once?

Reply:
Perhaps:
- publishing to a package repository, such as `PyPI` and `conda-forge`

Reply:
ok just coming back to this now. looking at this language we definitely would rather the review come to us prior to a journal article (altho not required) but most packages are already on pypi in our experience and we've been reviewing since 2019. we have one package that was only on bioconda (for very good reason because of dependencies). we could easily leave the text as is.

But the reality is that people often want to get the package name they selected. so they will push to pypi as a placeholder until the full package is done. should we adjust the. language to loosen up where it should be published first?

i also like published to a package repository such as PyPI or conda-forge as it leaves space for both / either. 

thoughts @hamogu @willingc ?

Reply:
Published to a package repository makes sense to me. @lwasser